examples/routing/mini_gamma_IRF.jl

```diff
@@ -30,35 +30,57 @@ forcing_timeseries_dir =
     joinpath(data_file_path, "timeseries", "timeseries_lv05")
 output_dir =
     joinpath(data_file_path, "simulations", "simulations_lv05", "gamma_IRF")
-@info "creating output path"
+@info "output path: $output_dir"
 if !isdir(output_dir)
     mkpath(output_dir)
 end
 
+# data information
+data_start_date = Date("1996-01-01", "yyyy-mm-dd")
+data_end_date = Date("2014-12-31", "yyyy-mm-dd") # of entire simulation
+data_step = Day(1)
+data_date_window = DateWindow(
```

Comment:
I think a great followon PR for us would be to make this just be of length t_max_hillslope, and to read in only the chunk of forcing data needed to compute hillslope state.

Reply:
This is just the data window to prescribe the beginning and end of a simulation, I'd like it not directly coupled to the river model (just because different models may use different histories etc.) But i agree the setup could be done better. 

But there is nothing to stop us from just reading in only the relevant dates and creating the envrionment of this length. I do think this could be refined